components/resources/speedreader_strings.grdp

```diff
@@ -5,22 +5,22 @@
     <message name="IDS_READER_MODE_MINUTES_TEXT" desc="Text of the read time element">min. read</message>
 
     <if expr="not is_android">
-      <message name="IDS_SPEEDREADER_ICON_TURN_OFF_READER_MODE" desc="Accessible text shown when a page is in Reader Mode">Turn off speedreader</message>
-      <message name="IDS_SPEEDREADER_ICON_TURN_ON_READER_MODE" desc="Accessible text shown when a page can be distilled">Turn on speedreader</message>
-      <message name="IDS_READER_MODE_ALWAYS_LOAD_FOR_SITE_LABEL" desc="Used for a toogble button label">Always use speedreader for this site</message>
-      <message name="IDS_READER_MODE_ALWAYS_LOAD_FOR_SITE_ACC" desc="Accessible text for the toggle to enable or disable speedreader per site">
-        Toggle to turn on/off speedreader for this website.
+      <message name="IDS_SPEEDREADER_ICON_TURN_OFF_READER_MODE" desc="Accessible text shown when a page is in Reader Mode">Turn off Speedreader</message>
+      <message name="IDS_SPEEDREADER_ICON_TURN_ON_READER_MODE" desc="Accessible text shown when a page can be distilled">Turn on Speedreader</message>
+      <message name="IDS_READER_MODE_ALWAYS_LOAD_FOR_SITE_LABEL" desc="Used for a tooglable button label">Always use Speedreader for this site</message>
```

Comment:
I think it meant to say `a toggle button`

Reply:
Fixed - Changed to toggle as required - not completely sure though about the wording as 'toggleable' is not really a word but can be used as one.